backend/src/services/words/words.service.ts

```diff
@@ -0,0 +1,11 @@
+import { postgres } from "../../utils/database/connect-to-database";
+
+export class WordsService {
+  static async getWords(owner_id: number, set_id: number) {
+    const words = await postgres
+      .select("original_word", "translated_word")
+      .from("words")
+      .where({ owner_id: owner_id, set_id: set_id });
```

Comment:
have you checked if this config is protecting your database from [SQL Injection](https://www.w3schools.com/sql/sql_injection.asp)? I think knex supports this out of the box but it might be a learning experience to check for this, you can call your routes a couple times with common phrases that'd allow the requester to manipulate your database